Helpers/Eligibility.php

```diff
@@ -182,6 +188,7 @@ public function getPlansEligibility(): array
      */
     public function checkEligibility()
     {
+        // TODO : Do not check Eligibility when cart is empty
```

Comment:
is it the `checkEligibility` responsibility or the method that will call it ?

put the todo in the doc block ;)